src/version.py

```
@@ -0,0 +1,43 @@
+
+
+def get_version_safe() -> str:
+    """
+    Get the package version with safe fallback.
+    
+    Returns:
+        str: The version string, with fallback to development version
+    """
+    try:
+        return get_version()
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)